ivre/db/__init__.py

```diff
@@ -1498,6 +1498,51 @@ def searchtorcert(self):
     def searchcertsubject(expr):
         raise NotImplementedError
 
+    @staticmethod
+    def searchdns(name, reverse=False, subdomains=False):
+        """Filters DNS records for domain `name`.
+        `name` can be a string, a list or a regular expression.
+        If `reverse` is set to True, filters reverse records.
+        If `subdomains` is set to True, the filter will match any subdomains.
+        """
+        raise NotImplementedError
+
+    def get(self, spec, **kargs):
+        """Queries the active column with the provided filter "spec",
```

Comment:
Could you remove the references to MongoDB / fast cursor / etc.?, since this part is backend-independent.

Reply:
This comment still applies.